var/spack/repos/builtin/packages/darshan-runtime/package.py

```diff
@@ -34,9 +34,9 @@ class DarshanRuntime(Package):
     systems where you intend to instrument MPI applications."""
 
     homepage = "http://www.mcs.anl.gov/research/projects/darshan/"
-    url = "http://ftp.mcs.anl.gov/pub/darshan/releases/darshan-3.1.0.tar.gz"
+    url = "https://xgitlab.cels.anl.gov/darshan/darshan/repository/darshan-3.1.6/archive.tar.gz"
```

Comment:
Why the URL change? Have any of the old checksums changed as well?

Reply:
It wasn't working as explained [here](https://www.mcs.anl.gov/research/projects/darshan/2018/03/29/ftp-download-problems/)

Reply:
Also, the mcs site is very unreliable half the time in my experience.